odata-protocol/10 Context URL.md

```diff
@@ -176,7 +187,7 @@ Context URL template:
 
     {context-url}#{entity-set}{/type-name}/$entity
```

Comment:
This section and the preceding one only applies if the request has a type-cast segment